build/azure-pipelines/sql-product-compile.yml

```diff
@@ -102,15 +93,14 @@ steps:
     yarn gulp vscode-reh-linux-x64-min
     yarn gulp vscode-reh-web-linux-x64-min
   displayName: Compile
-  condition: and(succeeded(), ne(variables['CacheRestored-Compilation'], 'true'))
 
 - script: |
     set -e
     AZURE_STORAGE_ACCOUNT="$(sourcemap-storage-account)" \
     AZURE_STORAGE_ACCESS_KEY="$(sourcemap-storage-key)" \
       node build/azure-pipelines/upload-sourcemaps
   displayName: Upload sourcemaps
-  condition: and(succeeded(), ne(variables['CacheExists-Compilation'], 'true'))
+  condition: and(succeeded(), ne(variables['VSCODE_PUBLISH'], 'false'))
```

Comment:
This seems unrelated, why was this updated?

Reply:
I just updated the flags for this task from vscode while removing CacheExists-Compilation check